reporters/ClickhouseReporter.js

```diff
@@ -95,47 +95,55 @@ const TABLE_DEFINITIONS = [
     PRIMARY KEY(crawlId, pageId, targetId)`,
 ];
 
+/**
+ * @param {string | string[]} args
+ */
+function santizeCallArgs(args) {
+    // in some cases call args have been stringified, so unwrap that first.
+    const argsArray = typeof args === 'string'  ? JSON.parse(args) : (args || []);
+    return argsArray.map((/** @type {string} */ s) => s.replace(/'/g, ''));
```

Comment:
hm, what's going on in here? Why are we getting rid of `'`? because it can mess up clickhouse query? Isn't clickhouse library providing a sanitization function?

Reply:
Yes - it seems the library doesn't handle escaping of strings in array types, so I had to do it myself here.